storage/ios/Plugin/Storage.swift

```diff
@@ -0,0 +1,64 @@
+import Foundation
+
+public struct StorageConfiguration {
+    enum Group {
+        case named(String), cordovaNativeStorage
+    }
+
+    let group: Group
+
+    init(for group: Group = .named("CapacitorStorage")) {
+        self.group = group
+    }
+}
+
+public class Storage: NSObject {
```

Comment:
The `NSObject` should be removed. Swift structs (and enums with associated values) cannot be bridged to Obj-C so this class's designated initializer cannot be called from Obj-C code. As such, it's better to hide this class entirely and only make it available to Swift code.
```suggestion
public class Storage {
```

Reply:
Thanks, I think I saw that somewhere on the internet and wondered why I needed to do it